atoma-proxy/src/server/streamer.rs

```diff
@@ -105,9 +110,11 @@ impl Streamer {
     pub fn new(
```

Comment:
Update the docs to explicitly mention that one of stack small id or fiat estimated amount needs to be provided.